apps/src/templates/teacherDashboard/sections.js

```diff
@@ -77,13 +79,24 @@ export function renderLoginTypeAndSharingControls(sectionId) {
   );
 }
 
-export function renderSectionTable(sectionId, loginType) {
+export function renderSectionTable(sectionId, loginType, courseId) {
   registerReducers({manageStudents, isRtl});
   const store = getStore();
 
   store.dispatch(setLoginType(loginType));
   store.dispatch(setSectionId(sectionId));
 
+  const showShareColumn = experiments.isEnabled(experiments.SHARE_COLUMN);
+
+  // Show share column by default for CSD and CSP courses
+  const coursesToShowShareSetting = {
+    csd: 17,
+    csp: 18,
```

Comment:
are these the course ids from your local database? if so, I'm not sure this will work in production, where their ids are as follows:
```
| csd                                    | 14 |
| csp                                    | 15 |
```
ids are different again in test and other environments. ideally, we would identify the courses by name. it may be as simple as adding course_name here: https://github.com/code-dot-org/code-dot-org/blob/2649f149526dfba1530b1675e6559991133e1941/pegasus/helpers/section_api_helpers.rb#L647-L650

Reply:
ohh good call, will fix!